yarn.lock

```diff
@@ -7307,9 +7307,9 @@ typedarray@^0.0.6:
   integrity sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=
 
 typescript@^3.2.1:
```

Comment:
This looks wrong?

Reply:
Oh yup, fixed. `yarn upgrade` didn't update `package.json`, then I bumped it manually.